### ISSUE TITLE
Scripts: Daily scrape does not appear to be populating staging database

### ISSUE THREAD

**Issue by hancush** [datamade/la-metro-dashboard#58]

**Comment by hancush** [comment]
When I looked at the staging site, this week's events did not have the agendas posted Friday. This is curious, because the logs indicate that the imports from the full scrape occurred last night:

```
[2020-09-22 04:00:26,472] {docker.py:258} INFO - import:
[2020-09-22 04:00:26,472] {docker.py:258} INFO - bill: 0 new 0 updated 221 noop
[2020-09-22 04:00:26,472] {docker.py:258} INFO - event: 0 new 0 updated 428 noop
[2020-09-22 04:00:26,472] {docker.py:258} INFO - jurisdiction: 0 new 0 updated 1 noop
[2020-09-22 04:00:26,472] {docker.py:258} INFO - membership: 0 new 0 updated 514 noop
[2020-09-22 04:00:26,472] {docker.py:258} INFO - organization: 0 new 0 updated 18 noop
[2020-09-22 04:00:26,472] {docker.py:258} INFO - person: 0 new 0 updated 43 noop
[2020-09-22 04:00:26,473] {docker.py:258} INFO - post: 0 new 0 updated 18 noop
[2020-09-22 04:00:26,473] {docker.py:258} INFO - vote_event: 0 new 0 updated 77 noop
[2020-09-22 04:00:26,946] {docker.py:258} INFO - Importing into staging database postgres://datamade@boardagendas.metro.net:5432/lametro_staging
[2020-09-22 04:00:29,229] {docker.py:258} INFO - lametro (import)
[2020-09-22 04:00:29,230] {docker.py:258} INFO - bills: {}
[2020-09-22 04:00:29,230] {docker.py:258} INFO - people: {}
[2020-09-22 04:00:29,230] {docker.py:258} INFO - events: {}
[2020-09-22 04:00:29,245] {docker.py:258} INFO - import jurisdictions...
[2020-09-22 04:00:29,290] {docker.py:258} INFO - import organizations...
[2020-09-22 04:00:29,464] {docker.py:258} INFO - import people...
[2020-09-22 04:00:29,946] {docker.py:258} INFO - import posts...
[2020-09-22 04:00:30,033] {docker.py:258} INFO - import memberships...
[2020-09-22 04:00:33,245] {docker.py:258} INFO - import bills...
--
import:
[2020-09-22 04:01:49,384] {docker.py:258} INFO - bill: 0 new 0 updated 221 noop
  event: 0 new 0 updated 428 noop
  jurisdiction: 0 new 0 updated 1 noop
  membership: 0 new 0 updated 514 noop
  organization: 0 new 0 updated 18 noop
  person: 0 new 0 updated 43 noop
  post: 0 new 0 updated 18 noop
  vote_event: 0 new 0 updated 77 noop
--
[2020-09-22 11:32:53,688] {docker.py:258} INFO - import:
[2020-09-22 11:32:53,688] {docker.py:258} INFO - bill: 0 new 0 updated 3366 noop
[2020-09-22 11:32:53,688] {docker.py:258} INFO - jurisdiction: 0 new 0 updated 1 noop
[2020-09-22 11:32:53,688] {docker.py:258} INFO - organization: 0 new 0 updated 3 noop
[2020-09-22 11:32:53,688] {docker.py:258} INFO - post: 0 new 0 updated 18 noop
[2020-09-22 11:32:53,688] {docker.py:258} INFO - vote_event: 0 new 0 updated 1584 noop
[2020-09-22 11:32:54,071] {docker.py:258} INFO - Importing into staging database postgres://datamade@boardagendas.metro.net:5432/lametro_staging
[2020-09-22 11:32:55,776] {docker.py:258} INFO - lametro (import)
  bills: {}
  people: {}
  events: {}
[2020-09-22 11:32:55,792] {docker.py:258} INFO - import jurisdictions...
[2020-09-22 11:32:55,816] {docker.py:258} INFO - import organizations...
[2020-09-22 11:32:55,853] {docker.py:258} INFO - import people...
[2020-09-22 11:32:55,861] {docker.py:258} INFO - import posts...
[2020-09-22 11:32:55,935] {docker.py:258} INFO - import memberships...
[2020-09-22 11:32:55,943] {docker.py:258} INFO - import bills...
[2020-09-22 11:34:18,963] {docker.py:258} INFO - import events...
[2020-09-22 11:34:18,974] {docker.py:258} INFO - import vote events...
[2020-09-22 11:34:34,251] {docker.py:258} INFO - lametro (import)
  bills: {}
  people: {}
  events: {}
import:
  bill: 0 new 0 updated 3366 noop
  jurisdiction: 0 new 0 updated 1 noop
  organization: 0 new 0 updated 3 noop
  post: 0 new 0 updated 18 noop
  vote_event: 0 new 0 updated 1584 noop
[2020-09-22 11:34:35,369] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=daily_scraping, task_id=daily_scraping, execution_date=20200921T030500, start_date=20200922T030617, end_date=20200922T113435
[2020-09-22 11:34:36,040] {logging_mixin.py:112} INFO - [2020-09-22 11:34:36,040] {local_task_job.py:103} INFO - Task exited with return code 0
```

**Comment by hancush** [comment]
Strikes me that the entity counts in the import reports are the same. The counts in the staging / live database are not:

```
ubuntu@ip-10-0-0-48:~$ psql -U postgres -d lametro_staging
psql (10.14 (Ubuntu 10.14-0ubuntu0.18.04.1))
Type "help" for help.

lametro_staging=# select count(*) from opencivicdata_bill;
 count
-------
  3363
(1 row)

lametro_staging=# \q
ubuntu@ip-10-0-0-48:~$ psql -U postgres -d lametro
psql (10.14 (Ubuntu 10.14-0ubuntu0.18.04.1))
Type "help" for help.

lametro=# select count(*) from opencivicdata_bill;
 count
-------
  3372
(1 row)
```

So, I think this is actually running the import against the production database twice. Probably something to do with pupa not using the DATABASE_URL we're passing when we run the import command.

https://github.com/datamade/la-metro-dashboard/blob/ac416e5e03f6a97fb9b0c6112093c679cefb0d1c/dags/scripts/full-scrape.sh#L4-L9

**Comment by hancush** [comment]
We [append a value for `DATABASE_URL`](https://github.com/datamade/la-metro-dashboard/blob/a0731b86eb6b4b681d4078b686888663e34d2130/scripts/concat_settings.sh#L20) to the `pupa_settings.py` file that [sets the URL from the `LA_METRO_DATABASE_URL`](https://github.com/datamade/la-metro-dashboard/blob/a0731b86eb6b4b681d4078b686888663e34d2130/configs/connection_settings.py#L8), which is threaded into the app environment through Supervisor, then [passed to the Docker container](https://github.com/datamade/la-metro-dashboard/blob/a0731b86eb6b4b681d4078b686888663e34d2130/dags/scrape_factory.py#L15) for each task. 

This was overriding the original definition, which sources from `DATABASE_URL`. 

Two-step solution:

- [x] Set the `LA_METRO_DATABASE_URL` environment variable in the conditional import function
- [ ] Pass a value for `LA_METRO_STAGING_DATABASE_URL` to the Docker environment

**Comment by hancush** [comment]
Resolved in #68 and deployed! All staging scrapers officially turned off.
